config/auth_permutations_test.go

```diff
@@ -232,6 +234,50 @@ func TestConfig_PatFromDatabricksCfg_NohostProfile(t *testing.T) {
 	}.apply(t)
 }
 
+func TestConfig_DatabricksCfgProfileResolve_ThroughHost(t *testing.T) {
+	configFixture{
+		loaders: []Loader{
+			ConfigAttributes,
+			ImplicitProfileLoader{},
+		},
+		env: map[string]string{
+			"HOME":            "testdata",
+			"DATABRICKS_HOST": "https://dbc-XXXXXXXX-ABCD.cloud.databricks.com",
+		},
+		assertHost: "https://dbc-XXXXXXXX-ABCD.cloud.databricks.com",
+		assertAuth: "basic",
+	}.apply(t)
+}
+
+func TestConfig_DatabricksCfgProfileResolve_ThroughHost_WithToken(t *testing.T) {
+	configFixture{
+		loaders: []Loader{
+			ConfigAttributes,
+			ImplicitProfileLoader{},
+		},
+		env: map[string]string{
+			"HOME":             "testdata",
+			"DATABRICKS_HOST":  "https://dbc-XXXXXXXX-ABCD.cloud.databricks.com",
+			"DATABRICKS_TOKEN": "abcd",
+		},
+		assertError: "validate: more than one authorization method configured: basic and pat. Config: host=https://dbc-XXXXXXXX-ABCD.cloud.databricks.com, token=***, username=abc, password=***, profile=basic. Env: DATABRICKS_HOST, DATABRICKS_TOKEN",
```

Comment:
This shouldn't fail. It is clear that the token should be used regardless in this case.